Feat/create landing page

tordjoha opened the pull request:
Added:
- landing page containers and content, some placeholders
- basic footer - change this to own component in later build

Changed:
- first nav bar entry to 'home'
- removed commented out css code in frontend/app/page.module.css
- centered the create tournament form on the website

TODO:
- make buttons in sign up, create and create tournament active
- change footer to own element like the navbar
- placeholder logo

Ivhene commented on frontend/components/createTournament/CreateTournamentViewStyle.module.css:
burde kanskje lage en ny styles mappe for css, components burde bare være UI komponenter

Ivhene commented on frontend/public/arrow-down.svg:
https://www.radix-ui.com/icons Kanskje bruke disse i stedet for å ha bilder av ikoner

Ivhene reviewed (changes requested):
se kommentar på arrow-down.svg og CreateTournamentViewStyle

Footer på main page kan kanskje flyttes til root layout.tsx, kan være grei å ha på alle sider

tordjoha commented:
Changes after review: 
- added icons from radix ui instead of random svg.
- made footer a component
- made styles folder for all css
- fixed import statements for css
